synthtool/gcp/templates/python_library/.kokoro/publish-docs.sh

```diff
@@ -18,26 +18,16 @@ set -eo pipefail
 # Disable buffering, so that the logs stream through.
 export PYTHONUNBUFFERED=1
 
-cd github/{{ metadata['repo']['repo'].split('/')[1] }}
```

Comment:
Does trampoline handle `cd`ing into the directory?

Reply:
yes